packages/webapp/src/components/lists/ReportList/index.tsx

```diff
@@ -323,6 +331,26 @@ const ReportList = memo(function ReportList({ title }: ReportListProps): JSX.Ele
 	// #endregion Report filter functions
 
 	// #region Service Report functions
+	const handleUpdateServiceAnswer = async (values) => {
```

Comment:
This is fine, but as a minor point, `function`s are probably the right choice over lambas for functions of this size. They provide call-stack information via their function name, where lambdas are anonymous in call stacks. Personally I only use lambdas for tiny callbacks or when I'm being deliberate about binding the `this` context.

Reply:
applied code review